docs/data-tunnel/route-types/01-tss.md

```diff
@@ -1,13 +1,81 @@
 # TSS Route
 
-:::warning
+## Overview
 
-We are excited to announce that the Band v3 testnet is currently under development!
+The **TSS Route** is one of the tunnel routes used to transmit tunnel packet data from **BandChain** to other chains (currently supporting only EVM-compatible chains). This enables developers to create smart contracts that receive real-time price data from BandChain and seamlessly integrate it into their applications by utilizing both BandChain price data and Tunnel Router bridge.
 
```

Comment:
may need to add some link to https://github.com/bandprotocol/tunnel-tss-router-contracts/blob/main/docs/contracts/02-dataflow.md. so user can understand overall on the contract side as well